modules/installation-azure-service-principal.adoc

```diff
@@ -222,31 +220,29 @@ If you limit the service principal scope of the {product-title} installation pro
 [source,terminal]
 ----
 $ az role assignment create --role "User Access Administrator" \
-    --assignee-object-id $(az ad sp list --filter "appId eq '<appId>'" \
-       | jq '.[0].id' -r) <1>
+  --assignee-object-id $(az ad sp show --id --query Id -o tsv)
 ----
-<1> Replace `<appId>` with the `appId` parameter value for your service principal.
 
```

Comment:
$(az ad sp show --id **<appId>** --query **id** -o tsv)
need add the appId 
id not Id

Reply:
That's what I was asking in my previous comment.

Updated. Please review.